src/PartielsPy/export_configs/export_config_image.py

```diff
@@ -0,0 +1,111 @@
+"""A class for export image configuration, used for jpeg and png formats"""
+
+from enum import Enum
+
+from ._export_config import ExportConfig
+
+
+class ExportConfigImage(ExportConfig):
+    class FormatEnum(Enum):
+        """Enum for the image format"""
+
+        JPEG = "jpeg"
+        PNG = "png"
+
+    def __init__(
+        self,
+        format: str = FormatEnum.JPEG.value,
+        width: int = 1280,
+        height: int = 720,
+        groups: bool = False,
+    ):
+        """Initialize the export configuration for the image.
+
+        This class is used to configure the export of the image.
+        Images format is jpeg or png.
+        The default format is jpeg.
+        The default width is 1280 pixels.
+        The default height is 720 pixels.
+        The default groups option is False.
+
+        Args:
+            format (str): the format of the image (jpeg or png)
+            width (int): the width of the image in pixels
+            height (int): the height of the image in pixels
+            groups (bool): if True exports the images of group and not the image
```

Comment:
The name is not very explicit. You can refer as the one used in the GUI of Partiels.